prefork/prefork.go

```diff
@@ -0,0 +1,179 @@
+package prefork
+
+import (
+	"flag"
+	"net"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/valyala/fasthttp"
+	"github.com/valyala/fasthttp/reuseport"
+)
+
+const preforkChildFlag = "prefork-child"
+const defaultNetwork = "tcp4"
+
+var Child bool
+
+// Prefork implements fasthttp server prefork
+//
+// Preforks master process (with all cores) between several child processes
+// increases performance significantly, because Go doesn't have to share
+// and manage memory between cores
+//
+// WARNING: Does not recommended for servers with in-memory cache,
+// because the cache will be duplicated in each process
```

Comment:
Please change this warning to something like:
```
WARNING: using prefork prevents the use of any global state! Things like in-memory caches won't work.
```
It's not just about in-memory caches. Everything that uses global state won't work.

Reply:
Done!